services/state/schedule/schedule.proto

```diff
@@ -71,19 +71,19 @@ message Schedule {
     /**
      * This schedule's unique ID within the global network state.
      */
-    ScheduleID id = 5;
+    ScheduleID schedule_id = 5;
 
     /**
      * The schedule account for this schedule.  This is the account that submitted the original
      * ScheduleCreate transaction.
      */
-    AccountID scheduler_account = 6;
+    AccountID scheduler_account_id = 6;
 
     /**
      * The explicit payer account for the scheduled transaction.
      * This account is added to the accounts that must sign the schedule before it will execute.
      */
-    AccountID payer_account = 7;
+    AccountID payer_account_id = 7;
```

Comment:
Why add the unnecessary and redundant `_id` value here (and for scheduler_account above)?

Reply:
In this case, I think the _id is good because it refers to an AccountID rather than an Account.